scripts/imageIDI.py

```diff
@@ -235,7 +235,7 @@ def main(args):
                 if yn not in ('n', 'N'):
                     overwrite = True
             try:
-                hdulist.writeto(args.fits, overwrite=overwrite)
+                hdulist.writeto(f"{args.fits}-t{set}.fits", overwrite=overwrite)
```

Comment:
1. There's a line above this where it checks for the existence of a file called `args.fits` that needs to be updated as well.
 2. I haven't officially dropped support for Py2 so we can't assume f-string support.

I'll take care of both of these.